src/main/java/org/fogbowcloud/saps/engine/core/service/ObjectStoreService.java

```diff
@@ -0,0 +1,114 @@
+package org.fogbowcloud.saps.engine.core.service;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.apache.log4j.Logger;
+import org.fogbowcloud.manager.core.plugins.identity.openstack.KeystoneV3IdentityPlugin;
+import org.fogbowcloud.manager.occi.model.Token;
+import org.fogbowcloud.saps.engine.core.model.ImageTask;
+import org.fogbowcloud.saps.engine.scheduler.util.SapsPropertiesConstants;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+
+/**
+ * Service that provides operations for communication with Object Store.
+ */
+public class ObjectStoreService {
+
```

Comment:
this class seems very coupled with SWIFT logic. I think it would be better to call it SwiftStorageService (or something like that). overall, I think it is a very nice idea to hide the complexity.
Also, why no passing the properties file to a constructor as a regular object instead of the more staless design

Reply:
Yes, it is, I'll change it.

About the properties file, it's because that conversation we had about services of restlet's architecture, but because of the deadline I couldn't use them. So, my strategy was to simplify the utilization of services, making all of its methods as static, in order to avoid instantiate the services. 

The expected strategy was to create an interface to define the contract of the services and the instantiations of its implementations be done by injection (or some like that, I don't know how restlet handle this)

I'm open for suggestions, should I change the methods to be instance's and instantiate the services in constructors of its users?